charts/calico/templates/calico-typha.yaml

```diff
@@ -71,6 +71,11 @@ spec:
         # Mark the pod as a critical add-on for rescheduling.
         - key: CriticalAddonsOnly
           operator: Exists
+        # this taint is set by all kubelets running `--cloud-provider=external`
+        # so we should tolerate it to schedule typha pods
+        - key: node.cloudprovider.kubernetes.io/uninitialized
+          value: "true"
+          effect: NoSchedule
```

Comment:
Perhaps just copy the Tolerations from the calico-node pod (since calico-node cannot go ready until typha comes up): https://github.com/projectcalico/calico/blob/ddbc8e8141c811422cb4bcf8af177b99ac5fb82e/charts/calico/templates/calico-node.yaml#L31-L39

Reply:
```suggestion
         # Make sure Typha can get scheduled on any nodes. 
         - effect: NoSchedule 
           operator: Exists 
         - effect: NoExecute 
           operator: Exists           
```

Reply:
+1 on this, it seems like we already do that for typha when using operator: https://github.com/tigera/operator/blob/master/pkg/render/typha.go#L400